github/resource_github_repository_tag_protection.go

```diff
@@ -0,0 +1,118 @@
+package github
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"net/http"
+	"strconv"
+	"strings"
+
+	"github.com/google/go-github/v47/github"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+)
+
+func resourceGithubRepositoryTagProtection() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceGithubRepositoryTagProtectionCreateOrUpdate,
+		Read:   resourceGithubRepositoryTagProtectionRead,
+		//Update: resourceGithubRepositoryTagProtectionCreateOrUpdate,
```

Comment:
What's the reason this is commented out?

Reply:
I was testing adding this function and just missed this in my code cleanup 😅.